Cargo.toml

```diff
@@ -21,7 +21,7 @@ features = ["handleapi", "ws2def", "ws2ipdef", "ws2tcpip", "minwindef"]
 
 [target."cfg(any(unix, target_os = \"redox\"))".dependencies]
 cfg-if = "0.1.6"
-libc = "0.2.66"
+libc = { version = "0.2.66", features = ["align"] }
```

Comment:
The added feature is so that `libc::sockaddr_in6` does not have an extra `__align` field (used before `#[repr(align(x))]` was a thing.)